packages/idea-transformer/src/Transformer.ts

```diff
@@ -25,8 +25,14 @@ export default class Transformer<T extends Record<string, unknown>> {
       if (!fs.existsSync(this.input)) {
         throw Exception.for('Input file %s does not exist', this.input);
       }
-      //parse schema
-      const schema = parse(fs.readFileSync(this.input, 'utf8'));
+      //  read input file
+      const content = fs.readFileSync(this.input, 'utf8');
+
+      // parse schema
+      const schema: SchemaConfig = path.extname(this.input) === '.json'
```

Comment:
follow commenting pattern in file. Avoid lateral commenting. Commenting should be above the code in question. ie.

```js
//parse as normal
: parse(content); 
```